src/schemaToPropTypes.js

```diff
@@ -185,11 +185,34 @@ const schemasReducer = (str, [schemaName, schema]) => {
  */
 const generatePropTypes = (api, schemaToParse) => {
```

Comment:
Well… this is a good point, yet I'd keep it, because SmartBear donated the specification and accepted it to be named so:

> One of the most notable reasons why the release is so important is that OpenAPI 3.0 is the first official release of the specification since it was donated to the OpenAPI Initiative by SmartBear Software and renamed from the Swagger Specification to OpenAPI specification in 2015.

https://swagger.io/blog/api-strategy/difference-between-swagger-and-openapi/

This doesn't mean swagger 2 version has been renamed, but still the standard is now called openapi. Let's further consider this.

Reply:
Done